src/Materials/shaderMaterial.ts

```diff
@@ -718,6 +718,17 @@ export class ShaderMaterial extends Material {
             defines.push("#define NUM_MORPH_INFLUENCERS 0");
         }
 
+        // Baked Vertex Animation
+        if (mesh) {
+            const bvaManager = (<Mesh>mesh).bakedVertexAnimationManager;
+
+            if (bvaManager && bvaManager.isEnabled) {
+                defines.push("#define BAKED_VERTEX_ANIMATION_TEXTURE");

```

Comment:
small nit - no need to add "#define", it is now added automatically (if not present)